app/src/main/kotlin/com/saltedge/authenticator/features/actions/SubmitActionViewModel.kt

```diff
@@ -163,6 +170,50 @@ class SubmitActionViewModel(
             if (errorMessage.isEmpty()) appContext.getString(R.string.action_error_description) else errorMessage
         )
     }
+
+    private fun showConnectionsSelector(connections: List<Connection>) {
+        val result = connections.convertConnectionsToViewModels(
+            context = appContext,
+            locationManager = locationManager
+        )
+        viewMode = ViewMode.SELECT
+        showConnectionsSelectorFragmentEvent.postValue(ViewModelEvent(dataBundle(result)))
+    }
+
+    private fun openReturnToUrl() {
```

Comment:
it is not clear what the function does, judging by the name of the event, it opens a link.

Reply:
it tries to open return_to url

Reply:
maybe still open the link?
And in the description of the method to describe what exactly you want to open.

Reply:
name `openReturnToUrl` is enough